charts/catalog/templates/rbac.yaml

```diff
@@ -192,4 +192,30 @@ items:
     kind: ServiceAccount
     name: "{{ .Values.controllerManager.serviceAccount }}"
     namespace: "{{ .Release.Namespace }}"
+
+# This allows anyone to get the Service Catalog readiness probe
```

Comment:
Not sure if this is OK or not. Usually, people like to prevent everyone from getting the readiness probe (in case the readiness probe returns some kind of sensitive data).

Reply:
/healthz (liveness) is (& was before) open to anonymous

Reply:
I thought so. I guess it's fine.